coral/src/domain/schema-request/schema-request-types.ts

```diff
@@ -0,0 +1,5 @@
+import { KlawApiModel } from "types/utils";
+
+type SchemaRequest = KlawApiModel<"SchemaRequest">;
+
+export type { SchemaRequest };
```

Comment:
Is this missing the types for `KlawApiResponse<"schemaRegEnvsGet">` and `KlawApiResponse<"schemaUpload">` or would they be implemented later ?

Reply:
`KlawApiResponse<"schemaRegEnvsGet">` and `KlawApiResponse<"schemaUpload">` are only used inside the `/domain` directory right now. `schemaUpload` returns a list of `Environments` (we already have a type for that) and `schemaUpload` returns a lost of `SchemaRequest`. I don't (currently) see that we need types for that outside of `/domain`, so did not add them for now.

Reply:
Thank you for the explanation ^^